pyvrp/Statistics.py

```diff
@@ -164,14 +119,15 @@ def from_csv(cls, where: Path | str, delimiter: str = ",", **kwargs):
         """
         field2type = {field.name: field.type for field in fields(_Datum)}
 
-        def make_datum(row, prefix) -> _Datum:
+        def make_datum(row) -> _Datum:
             datum = {}
 
             for name, value in row.items():
-                if (field_name := name[len(prefix) :]) in field2type:
-                    # If the prefixless name is a field name, cast the row's
-                    # value to the appropriate type and add the data.
-                    datum[field_name] = field2type[field_name](value)
+                if name in field2type:
+                    if field2type[name] is bool:
+                        datum[name] = value.lower() == "true"
```

Comment:
This does not seem to be a robust way to handle booleans, but I wasn't quite sure what else to do. Store as 0/1s instead?